library/oneview_hypervisor_cluster_profile.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2020) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_hypervisor_cluster_profile
+short_description: Manage OneView Hypervisor Cluster Profiles resources.
+description:
+    - Provides an interface to manage Hypervisor Cluster Profiles resources. Can create, update, and delete.
+version_added: "2.4"
+requirements:
+    - "python >= 3.4.2"
+    - "hpOneView >= 5.1.0"
```

Comment:
hpOneView version should be 5.1.1

Reply:
actually i just noticed that python sdk patch is for your other PR#494. Please modify there.